### PR TITLE
Reveal information about container capabilities

### PR DESCRIPTION
**Pull request by rhatdan**
I am often asked about the list of capabilities availabel to a container.
We should be listing this data in the inspect command for effective
capabilities and the bounding set.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by vrothberg** [comment]
LGTM (tested locally)

File: docs/podman-inspect.1.md [hunk 1]
```diff
@@ -96,6 +96,11 @@ overlay
 size:   4405240
 ```
 
+```
+podman inspect --latest --format {{.EffectiveCaps}}
+[CAP_CHOWN CAP_DAC_OVERRIDE CAP_FSETID CAP_FOWNER CAP_MKNOD CAP_NET_RAW CAP_SETGID CAP_SETUID CAP_SETFCAP CAP_SETPCAP CAP_NET_BIND_SERVICE CAP_SYS_CHROOT CAP_KILL CAP_AUDIT_WRITE]
+```
+
```
Comment (TomSweeneyRedHat) [inline docs/podman-inspect.1.md]: The example at line 38 doesn't show these fields.  Can we update that?   Being careful to keep the other examples valid in comparison.

**Comment by TomSweeneyRedHat** [comment]
Otherwise LGTM

**Comment by rhatdan** [comment]
I don't think we should change that output based on this PR, we can visit this later in a different PR>

**Comment by rhatdan** [comment]
@rh-atomic-bot r=TomSweeneyRedHat

**Pull request closed without merging**